src/hotspot/share/compiler/directivesParser.cpp

```diff
@@ -335,6 +336,15 @@ bool DirectivesParser::set_option_flag(JSON_TYPE t, JSON_VAL* v, const key* opti
           if (!valid) {
             error(VALUE_ERROR, "Unrecognized intrinsic detected in DisableIntrinsic: %s", validator.what());
           }
+        } else if (strncmp(option_key->name, "TraceAutoVectorization", 22) == 0) {
```

Comment:
Missing `#ifndef PRODUCT` and `#ifdef COMPILER2` for this and for `PrintIdealPhase`.

Reply:
done